Sources/NIO/Selector.swift

```diff
@@ -297,13 +297,13 @@ final class Selector<R: Registration> {
 
```

Comment:
please also remove all the changes in this file, I think Xcode's 'rename' function got properly confused here...